### ISSUE TITLE
Mojave Support?

### ISSUE THREAD

**Issue by galaxy-nomad** [umfana/FaviconBar#56]
Hello,
I installed the Mojave beta and was so jazzed to finally be able to see bookmark favicons against a dark backdrop in Safari. Unfortunately Safari gave me the stiff arm when installing FaviconBar and said non Gallery extensions were no longer supported or something to that affect. Will this extension be updated going forward or is High Sierra the end of the line?

**Comment by popcornhax** [comment]
What exactly did it say?  Did you try to install from GitHub or the Apple extensions site?  Do you have a screenshot?

I upgraded to Mojave, and FaviconBar remained - but haven't tried installing clean.

Apple is pushing extensions to the App Store, but App Store extensions can't add toolbars so that would kill FaviconBar.  I'm hoping we can still keep it available via GitHub

**Comment by galaxy-nomad** [comment]
Give me a sec. Let me boot to my external drive with Mojave and see what it says.

**Comment by galaxy-nomad** [comment]
Here you go!
<img width="800" alt="screen shot 2018-06-23 at 1 48 59 pm" src="https://user-images.githubusercontent.com/12664589/41813517-4fedf9da-76ec-11e8-9240-6b7e5f5323b9.png">

**Comment by galaxy-nomad** [comment]
I couldn't find it in the new Extensions/App Store so I tried to install directly from the GitHub download.

**Comment by popcornhax** [comment]
Thanks, I'll dig into it and see what options we have.  It seems like at some point Apple will kill off old school extensions, but hopefully - not today

**Comment by galaxy-nomad** [comment]
Yeah I moved back over to FireFox for my custom dark theme only to learn that they gutted their extension gallery with the Quantum update. So many great tweaks gone but many new developers are coming with replacements so it's not so bad. I'm sure Apple is even more of a pain in the rear end for Extension developers to be able to pass muster. Thanks again for providing this extension. Many happy months with Safari and FaviconBar. I was inspired to implemented round Favicons to my FireFox theme because of the feature you had in FaviconBar!

![screen shot 2018-06-23 at 1 58 38 pm](https://user-images.githubusercontent.com/12664589/41813593-9a8dd108-76ed-11e8-842b-6a187b293ccb.jpg)

**Comment by galaxy-nomad** [comment]
YOOOO! I found a workaround and have FaviconBar installed and looking as good as ever! Check this out!! https://georgegarside.com/blog/macos/install-any-safari-extension-macos-mojave/

**Comment by galaxy-nomad** [comment]
One issue I'm having is I can't get the FaviconBar text white. Do you know the CSS code I can enter to make the text white?
<img width="1399" alt="screen shot 2018-06-24 at 6 30 15 pm" src="https://user-images.githubusercontent.com/12664589/41826239-d89eed7a-77dc-11e8-9fe3-051b30e36711.png">

**Comment by galaxy-nomad** [comment]
Hello. Just checking in to see if you'll be updating this plug-in for Mojave. I'm on High Sierra and I noticed that the text becomes hard to read when scrolling with transparency enabled. (reduce transparency unchecked) Is there a CSS command that would disable the white drop shadow and also make the text thicker like the Favorites bar above in the attached image?

![screen shot 2018-08-18 at 11 58 16 pm](https://user-images.githubusercontent.com/12664589/44306402-0e214d80-a343-11e8-8dd8-3c37d1a50e5e.jpg)
